packages/devkit/src/tests/create-tree-with-empty-workspace.ts

```diff
@@ -4,7 +4,7 @@ import type { Tree } from '@nrwl/tao/src/shared/tree';
 /**
  * Creates a host for testing.
  */
-export function createTreeWithEmptyWorkspace(version = 1): Tree {
+export function createTreeWithEmptyWorkspace(version = 1, nxConfig = {}): Tree {
```

Comment:
Please revert this change. This helper function is meant to create a standard empty workspace.

Instead, after creating the empty tree in your tests, do:

```typescript
updateJson(tree, 'nx.json', json => {
  // modify the json
  return json;
})
```

Reply:
done, seems good!